jime.csl

```diff
@@ -0,0 +1,362 @@
+        <text macro="date-issued-year"/>
+      </if>
+      <else-if variable="page" match="none">
+        <text macro="date-issued-year"/>
+      </else-if>
+      <else-if is-numeric="page">
+        <text macro="date-issued-year"/>
+      </else-if>
+      <else>
+        <text term="in press"/>
+      </else>
+    </choose>
+  </macro>
+  <macro name="date-issued-year">
+    <choose>
+      <if variable="issued">
+        <date date-parts="year" form="numeric" variable="issued">
+          <date-part name="year"/>
+        </date>
+      </if>
+      <else>
+        <text term="no date" form="short"/>
+      </else>
+    </choose>
+  </macro>
+  <macro name="date-access">
+    <date variable="accessed" prefix="[Last accessed " suffix="]">
+      <date-part name="day" form="numeric-leading-zeros" prefix=""/>
+      <date-part name="month" form="long" prefix=" "/>
+      <date-part name="year" prefix=" "/>
+    </date>
+  </macro>
+  <macro name="title">
+    <choose>
+      <if type="bill book graphic legal_case legislation motion_picture report song webpage broadcast" match="any">
+        <text variable="title" font-style="italic"/>
+      </if>
+      <else>
+        <text variable="title"/>
+      </else>
+    </choose>
+  </macro>
+  <macro name="book-details">
+    <group delimiter=". ">
+      <group delimiter=" ">
+        <number vertical-align="baseline" variable="edition" form="ordinal"/>
+        <label vertical-align="baseline" variable="edition" form="short"/>
+      </group>
+      <group delimiter=" ">
+        <label text-case="capitalize-first" variable="volume"/>
+        <text variable="volume"/>
+      </group>
+      <group font-variant="normal">
+        <text macro="editor-translator" font-variant="normal"/>
+      </group>
+      <group delimiter=". ">
+        <choose>
+          <if type="broadcast" match="any">
+            <text variable="number" prefix="Episode "/>
+          </if>
+        </choose>
+        <text variable="version"/>
+        <choose>
+          <if type="broadcast" match="none">
+            <text variable="medium"/>
+          </if>
+        </choose>
+      </group>
+      <group delimiter=" " prefix="(" suffix=")">
+        <text variable="collection-title"/>
+        <group delimiter=" ">
+          <label variable="issue" form="short"/>
+          <text variable="collection-number"/>
+        </group>
+      </group>
+      <choose>
+        <if type="paper-conference" match="any">
+          <date form="text" variable="issued"/>
+        </if>
+      </choose>
+      <text macro="publisher"/>
+    </group>
+  </macro>
+  <macro name="publisher">
+    <group>
+      <text variable="publisher-place" suffix=": "/>
+      <text variable="publisher"/>
+      <text variable="page" prefix=":"/>
+    </group>
+  </macro>
+  <macro name="access">
+    <choose>
+      <if type="book chapter thesis" match="none">
+        <choose>
+          <if variable="DOI">
```

Comment:
This can easily be simplified to the following as none of the groups do anything here and the DOI URI prefix should be closer to the variable anyway:
          <if variable="DOI">
                <text variable="DOI" prefix="https://doi.org/"/>
          </if>

Reply:
This logic came from the template CSL used to start this file. I didn’t see any provided example, but simplified the macro based on my understanding of what might be unnecessary via 926ef59.